crates/cxx-qt-lib/src/core/qpointf.rs

```diff
@@ -42,6 +42,24 @@ mod ffi {
         #[doc(hidden)]
         #[rust_name = "qpointf_to_qstring"]
         fn toQString(value: &QPointF) -> QString;
+        #[doc(hidden)]
+        #[rust_name = "qpointf_plus"]
+        fn operatorPlus(a: &QPointF, b: &QPointF) -> QPointF;
+        #[doc(hidden)]
+        #[rust_name = "qpointf_minus"]
+        fn operatorMinus(a: &QPointF, b: &QPointF) -> QPointF;
+        #[doc(hidden)]
+        #[rust_name = "qpointf_mul_f32"]
+        fn operatorMul(a: f32, b: &QPointF) -> QPointF;
+        #[doc(hidden)]
+        #[rust_name = "qpointf_mul_f64"]
+        fn operatorMul(a: f64, b: &QPointF) -> QPointF;
+        #[doc(hidden)]
+        #[rust_name = "qpointf_mul_i32"]
+        fn operatorMul(a: i32, b: &QPointF) -> QPointF;
+        #[doc(hidden)]
+        #[rust_name = "qpointf_div_f32"]
+        fn operatorDiv(a: f32, b: &QPointF) -> QPointF;
```

Comment:
note we need to be careful with `qreal`, sometimes between Qt 5 and Qt 6 it can be a `qreal` and then a `float` in the other etc or we have fun with 32bit platforms (eg Wasm).

Elsewhere we have a `f32` in rust, `float` in C++, then cast it to a `qreal` for the specific Qt version. So this looks potentially correct anyway :-)